config/prometheus/monitor.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     app.kubernetes.io/name: aibrix
     app.kubernetes.io/managed-by: kustomize
   name: controller-manager-metrics-monitor
-  namespace: system
```

Comment:
it will be overritten by kustomize if prometheus is under config/default. 

would this be applied separately or along with default?